resources/eds_templates/diff_report/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "template": "main.html",
+    "conf":
+    {
+        "default_endpoint" : "http://fuseki:3030/eurovoc-fragment/sparql",
```

Comment:
this cannot be hardcoded. endpooint is always variable./

Reply:
it is not hardcoded, it's the example I pulled from the `eds4jinja2`